packages/components/src/slot-fill/index.js

```diff
@@ -1,13 +1,28 @@
 /**
  * Internal dependencies
  */
-import Slot from './slot';
-import Fill from './fill';
-import Provider, { Consumer } from './context';
+import BaseSlot from './slot';
+import BaseFill from './fill';
```

Comment:
Should we remove all the bubblesVirtually logic from the BaseSlot, Baseffill and Provider?